src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -996,6 +996,45 @@ CUDNN_SAFE_CALL(cudnnSetOpTensorDescriptor(opTensorDesc,
                 writer.block_end();
             }
 
+            template <>
+            void GPU_Emitter::EMITTER_DECL(ngraph::op::FunctionCall)
+            {
+                auto function_call = static_cast<const ngraph::op::FunctionCall*>(node);
+                shared_ptr<Function> function = function_call->get_functions()[0];
+
+                writer.block_begin();
+                {
+                    vector<string> input_names;
```

Comment:
nit: can we use std:: for standard lib items? I prefer it, personally, and we do so already in gpu_emitter.cpp, so it would be good to be consistent.

Reply:
me too, I just did a copy and didn't change it.